airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -187,15 +187,29 @@ def _read(
         metadata['end_of_log'] = False if not logs else len(loading_hosts) == 0
 
         cur_ts = pendulum.now()
-        # Assume end of log after not receiving new log for 5 min,
-        # as executor heartbeat is 1 min and there might be some
-        # delay before Elasticsearch makes the log available.
         if 'last_log_timestamp' in metadata:
             last_log_ts = timezone.parse(metadata['last_log_timestamp'])
+
+            # if we are not getting any logs at all after more than N seconds of trying,
+            # assume logs do not exist
+            if int(next_offset) == 0 and cur_ts.diff(last_log_ts).in_seconds() > 5:
+                metadata['end_of_log'] = True
+                return [
+                    (
+                        '',
+                        (
+                            f"*** Log {log_id} not found in elasticsearch. "
```

Comment:
```suggestion
                            f"*** Log {log_id} not found in ElasticSearch. "
```

Reply:
just looked at elastic.co and they use `Elasticsearch`.  But we have `elasticsearch` in the wordlist and not `Elasticsearch`.  You reckon we should add `Elasticsearch` to the wordlist and change this?

Reply:
I think the wordlist is case insensitive so _Elastichsearch_ should just work?

Reply:
odd... if that were true i'd expect the wordlist to be all lowercase.... but ... let's find out!

Reply:
yeah i can find both lower and upper E elsewhere in this file.  made the change.